src/components/header/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FunctionComponent, useState } from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars, faTimes } from "@fortawesome/free-solid-svg-icons";
+import fsLogo from "../../assets/images/firstStepLogo.png";
+import useWindowDimensions from "../../hooks/useWindowDimensions";
+import styles from "./Header.module.scss";
+
+interface HeaderProps {}
+
+interface INav {
+  label: string;
+  url: string;
+}
+
+const Header: FunctionComponent<HeaderProps> = () => {
+  const [isOpen, useIsOpen] = useState(false);
+  const dimensions = useWindowDimensions();
+  const nav: Array<INav> = [
+    {
+      label: "Home",
+      url: "/home",
+    },
+    {
+      label: "About",
+      url: "/about",
+    },
+    {
+      label: "Contact Us",
+      url: "/Contactus",
+    },
+  ];
+  const linkType = isOpen ? styles.linkMobile : styles.linkDesktop;
+  const navList = nav.map((ctrl) => (
+    <Link className={linkType} key={ctrl.label} to={ctrl.url}>
+      {ctrl.label}
+    </Link>
+  ));
+
+  const sideBarStyles = isOpen ? styles.open : styles.close;
+  const navigation = (
+    <nav className={isOpen ? styles.listMobile : styles.listDesktop}>
+      {navList}
+    </nav>
+  );
+
+  return (
+    <header className={styles.header}>
+      <div className={`${styles.cont} container`}>
+        <div className={styles.content}>
+          <img className={styles.img} src={fsLogo} alt="" />
+          <h2 className={styles.title}>First Step Pharma</h2>
+        </div>
+        {dimensions.device === "desktop" ? (
+          navigation
+        ) : (
+          <FontAwesomeIcon
+            onClick={() => {
+              useIsOpen(true);
+            }}
+            icon={faBars}
+            className={`${styles.faBars} ${styles.content}`}
+          />
+        )}
+      </div>
+      <div className={sideBarStyles}>
```

Comment:
This section is not required on the desktop. Please put some condition to remove this for desktop.